python/python/glide/async_commands/server_modules/json.py

```diff
@@ -139,6 +139,55 @@ async def get(
     return cast(bytes, await client.custom_command(args))
 
 
+async def arrappend(
+    client: TGlideClient,
+    key: TEncodable,
+    values: List[TEncodable],
+    path: Optional[TEncodable] = None,
```

Comment:
`path` is not optional for JSON.ARRAPPEND. Ref doc: https://docs.aws.amazon.com/AmazonElastiCache/latest/dg/json-arrappend.html

Reply:
True, but it is optional is redis , wasn't sure what I'm supposed to do 🙁

Reply:
I think we should ignore redis impl for the module